template/manifests/PodDisruptionBudget/manifest/draft.yaml

```diff
@@ -0,0 +1,42 @@
+templateName: "podDisruptionBudget-manifest"
+description: "This template is used to create a PodDisruptionBudget for an application"
+type: "podDisruptionBudget"
+variables:
+  - name: "APPNAME"
+    type: "string"
+    kind: "resourceName"
```

Comment:
the `kind` params should follow the other draft.yaml definitions:
https://github.com/Azure/draft/blob/d45a77ac45e34416de33bd4579d733ce53cfd8e8/template/deployments/manifests/draft.yaml#L16C12-L16C34